recipes/_configure_server.rb

```diff
@@ -95,12 +96,46 @@
   end
 end
 
+path = "#{node['splunk']['home']}/etc/system/local/server.conf"
+old_stanzas = CernerSplunk::Conf::Reader.new(path).read if File.exist?(path)
```

Comment:
I think I see what you're saying, but right now we only need a value from this file in the case of an SHCluster. Not a big deal, but we're just reading a file unnecessarily for other cases.